### PR TITLE
remove dead code

### PR DESCRIPTION
**Pull request by benschac**
## What does this PR do and why?
ran `yarn find-deadcode` I removed a bunch of code and exports that aren't used anymore. I didn't remove code that's in `v2` with the assumption that the code that's not used is WIP (work in progress) and will be used shortly.
list of `deadcode` that's still in the project.
```
src/utils/featureFlags.ts:13 - enableFeatureFlag
src/utils/featureFlags.ts:17 - disableFeatureFlag
src/utils/featureFlags.ts:29 - featureFlagEnabled
src/utils/v2/currency.ts:44 - V2CurrencySymbol
src/utils/v2/currency.ts:49 - V2CurrencyStyle
src/utils/v2/currency.ts:52 - toV2Currency
src/components/v2/shared/V2SupportWarning.tsx:5 - default
src/hooks/v2/contractReader/TerminalCurrentReclaimableOverflow.ts:13 - default
src/components/v2/V2Create/forms/RulesForm/TokenMintingExtra.tsx:6 - default
src/components/v2/V2Project/V2ProjectReconfigureModal/drawers/V2ReconfigureFundingDrawer.tsx:6 - V2ReconfigureFundingDrawer
src/components/v2/V2Project/V2ProjectReconfigureModal/drawers/V2ReconfigureTokenDrawer.tsx:8 - V2ReconfigureTokenDrawer
```
cc: @aeolianeth @peripheralist @johnnyd-eth, should I remove any of the code listed above? I'm assuming since it's in v2 it's currently being built out and the unused code should be expected.

How I tested the changes in this PR:
- Went to `Router.tsx` and visited every top-level route to confirm there aren't any `null exception` errors that could possibly crash the app.
- Signed in with my eth wallet on `rinkeby` and did the above.
- Opened modals
- Made a test transaction in `http://localhost:3000/#/p/crewrevvheads`

## Acceptance checklist

- [x] I have evaluated the
      [Approval Guidelines](https://github.com/jbx-protocol/juice-interface/blob/main/CONTRIBUTING.md#approval-guidelines)
      for this PR.
- [x] I have tested this PR in
      [all supported browsers](https://github.com/jbx-protocol/juice-interface/blob/main/CONTRIBUTING.md#supported-browsers).

### CODE REVIEW

**Comment by benschac** [comment]
Current dependencies on/for this PR:
* main
  * **PR #1050** <a href="https://app.graphite.dev/github/pr/jbx-protocol/juice-interface/1050" target="_blank">Graphite</a> 
    * **PR #1049** <a href="https://app.graphite.dev/github/pr/jbx-protocol/juice-interface/1049" target="_blank">Graphite</a> 
    * **PR #1072** <a href="https://app.graphite.dev/github/pr/jbx-protocol/juice-interface/1072" target="_blank">Graphite</a> 
      * **PR #1073** <a href="https://app.graphite.dev/github/pr/jbx-protocol/juice-interface/1073" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/jbx-protocol/juice-interface/1073?utm_source=stack-comment).

File: src/locales/fr/messages.po [hunk 1]
```diff
@@ -3112,6 +3099,7 @@ msgstr "Votre solde"
 msgid "Your balance: {0}"
 msgstr "Votre solde : {0}"
 
+<<<<<<< HEAD
```
Comment (benschac) [inline src/locales/fr/messages.po:3102]: this needs to go. I'll mark this PR as blocked. Sorry everyone.

**Review by benschac — COMMENTED** [review]
Turns out I can't request changes on my own review. But, please ignore this PR until I fix the merge conflicts. Sorry about that everyone.

**Review by aeolianeth — APPROVED** [review]
@benschac sooo good!! My soul feels cleansed.

Merging.

**Pull request merged**